test/teos/e2e/test_basic_e2e.py

```diff
@@ -40,6 +40,9 @@
 
 teos_id, user_sk, user_id = teos_cli.load_keys(cli_config.get("TEOS_PUBLIC_KEY"), cli_config.get("CLI_PRIVATE_KEY"))
 
+appointment_in_watcher = 0
+appointment_in_responder = 0
```

Comment:
Should it be `appointments_in...` with an "s"?

Reply:
Agreed